action.yml

```diff
@@ -24,6 +24,14 @@ inputs:
     description: Boolean, replace/update existing translation on upload
     required: false
     default: false
+  apply-tm:
```

Comment:
Can you rename this to be more descriptive?
`apply-translation-matches`?

Reply:
Hmm, I guess it is localize's obscure name... 🤷🏾 
Never mind.

Reply:
tm is short for "translation memory". I named these parameters to be identical to the params/args for the Lokalise API.